src/Components/MovieForm/index.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { MovieFormAction, MovieFormProps } from './models'
+import './styles.scss'
+
+const MovieForm: React.FC<MovieFormProps> = ({ action, movie }) => {
+    return action === MovieFormAction.DELETE ? (
```

Comment:
this ternary can be rewritten. Cause for now you have 
```js
      <form className="movie-form">
            <div className="movie-form__container">
                <h2 className="movie-form__header">{action}</h2>
```
for both cases

Reply:
Refactored